### PR TITLE
Azure Cli extension for network-analytics data-product 

### PR DESCRIPTION
**Pull request by pikanghosh1**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/network-analytics/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
```
Comment (necusjz) [inline src/network-analytics/HISTORY.rst:6]:
```suggestion
1.0.0b1
```
Adapt to the upcoming versioning strategy.

File: src/network-analytics/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI NetworkAnalytics Extension #
+This is an extension to Azure CLI to manage NetworkAnalytics resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (necusjz) [inline src/network-analytics/README.md:5]: Please complete it with several examples.

File: src/network-analytics/azext_network_analytics/tests/user_identity.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{"/subscriptions/9a021c4c-ddcd-4e80-9155-a5b0e872f4c2/resourceGroups/a40rama/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ramatestiden":
+    {
+    "principalId": "00000000-0000-0000-0000-000000000000",
+    "clientId": "00000000-0000-0000-0000-000000000000"
+    }
+}
```
Comment (necusjz) [inline src/network-analytics/azext_network_analytics/tests/user_identity.json:6]: Please move this file under `latest` folder.

File: src/network-analytics/azext_network_analytics/tests/latest/test_network_analytics.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        self.cmd('az network-analytics data-product delete --name {name} --resource-group {group} --yes')
+
+        #wait for 20 second
+        time.sleep(20)
+
+        #List DP
+        dplistinrg = self.cmd('az network-analytics data-product list --resource-group {group}').get_output_in_json()
+        assert len(dplistinrg) == 0
+
+    pass
```
Comment (necusjz) [inline src/network-analytics/azext_network_analytics/tests/latest/test_network_analytics.py:75]: We shouldn't use `assert` for testing, our test sdk has its check method. You can refer to other modules' solution.

File: src/network-analytics/HISTORY.rst (hunk 1 above)
Comment (pikanghosh1) [reply src/network-analytics/HISTORY.rst]: version updated to 1.0.1

File: src/network-analytics/README.md (hunk 2 above)
Comment (pikanghosh1) [reply src/network-analytics/README.md]: Added examples for all the commands.

File: src/network-analytics/azext_network_analytics/tests/user_identity.json (hunk 3 above)
Comment (pikanghosh1) [reply src/network-analytics/azext_network_analytics/tests/user_identity.json]: Removed unused file.

File: src/network-analytics/azext_network_analytics/tests/latest/test_network_analytics.py (hunk 4 above)
Comment (pikanghosh1) [reply src/network-analytics/azext_network_analytics/tests/latest/test_network_analytics.py]: replaced assert with self.assertEquals and self.assertTrue

File: src/network-analytics/HISTORY.rst [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+1.0.1
```
Comment (necusjz) [inline src/network-analytics/HISTORY.rst:6]:
```suggestion
1.0.0b1
```

File: src/network-analytics/setup.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+
+# HISTORY.rst entry.
+VERSION = '1.0.1'
```
Comment (necusjz) [inline src/network-analytics/setup.py:13]:
```suggestion
VERSION = '1.0.0b1'
```

Comment (pikanghosh1) [reply src/network-analytics/setup.py]: revised version

File: src/network-analytics/HISTORY.rst (hunk 5 above)
Comment (pikanghosh1) [reply src/network-analytics/HISTORY.rst]: revised version

**Review by necusjz — APPROVED** [review]

**Pull request merged**